test/factbase/indexed/test_indexed_lt.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 Yegor Bugayenko
+# SPDX-License-Identifier: MIT
+
+require_relative '../../test__helper'
+require_relative '../../../lib/factbase'
+require_relative '../../../lib/factbase/term'
+require_relative '../../../lib/factbase/taped'
+require_relative '../../../lib/factbase/indexed/indexed_term'
+require_relative '../../../lib/factbase/indexed/indexed_lt'
+
+# Indexed term 'lt' test.
+# Author:: Yegor Bugayenko (yegor256@gmail.com)
+# Copyright:: Copyright (c) 2024-2025 Yegor Bugayenko
+# License:: MIT
+class TestIndexedLt < Factbase::Test
+  def test_predicts_on_lt
```

Comment:
@maxonfjvipon just moving the test is too little work for a single PR. Maybe you can add one more test here, for this method?